manifests/server/puppetserver.pp

```diff
@@ -232,4 +248,16 @@
     ensure  => $product_conf_ensure,
     content => template('puppet/server/puppetserver/conf.d/product.conf.erb'),
   }
+
+  # Do we want to leave this untouched for now?
+  # if versioncmp($server_puppetserver_version, '5.0') >= 0 {
```

Comment:
You mean possibilities for the `conf.d/metrics.conf` file? Or did I misunderstand?